docroot/themes/humsci/humsci_basic/tests/sass-specs/tools/_functions.animation.scss

```diff
@@ -0,0 +1,90 @@
+@include describe('HB Animation Function') {
+
+  // Colorful
+  @include describe('the hb-transition() function used with Colorful') {
+    $hb-current-theme: 'colorful' !global;
+
+    @include it('returns the theme variable values for the transition-duration and transition-timing-function properties for the Colorful theme') {
```

Comment:
might not need to say anything about the colorful theme again since you say it above!

you could just say "returns the proper theme values for transition-duration and transition-timing-function properties"